apps/desktop/electron-builder.json

```
@@ -256,6 +256,12 @@
           "action-prefix": "com.bitwarden.Bitwarden"
         }
       },
+      {
+        "personal-files": {
+          "read": [],
+          "write": ["$HOME/.config/chromium", "$HOME/.config/google-chrome", "$HOME/.mozilla"]
```

Could we reduce our permissions a bit more by just requiring the native messaging hosts folder?
```suggestion
          "write": ["$HOME/.config/chromium/NativeMessagingHosts", "$HOME/.config/google-chrome/NativeMessagingHosts", "$HOME/.mozilla/native-messaging-hosts"]
```